### PR TITLE
Add jupyter notebook used to explore activity detection results ove time

### PR DESCRIPTION
**Pull request by Purg**
This is a somewhat cleaned-up version of the notebook that was used to create plots for the site visit slides related to activity detector performance over time, with complete or abutting windows, and a proof-of-concept filter pass.

### CODE REVIEW

File: scripts/activity_detection_exploration/support_functions.py [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    bar_widths = [(p[1]-p[0]) for p in gt_ranges]
+    ax.bar(xs_bars, ys_gt_regions, width=bar_widths, align="edge", color="lightgreen")
+
+    if custom_range:
+        assert len(custom_range) == 2, "Assuming only two float values for custom range"
+        xs_bars2 = [custom_range[0]]
+        ys_height = [1.025] #[0.1]
+        bar_widths2 = [custom_range[1]-custom_range[0]]
+        ys_bottom = [0] #[1.01]
+        color = custom_range_color if custom_range_color is not None else "red"
```
Comment (hdefazio) [inline scripts/activity_detection_exploration/support_functions.py:132]: Can we just set the custom_range_color="red" in the function definition?

File: scripts/activity_detection_exploration/support_functions.py [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+    @property
+    def activity_predictions(self):
+        return GlobalValues.slice_preds[self.index]
+
+
+def plot_activity_confidence(label, gt_ranges, custom_range=None, custom_range_color=None):
+    """
+    Plot activity confidences, with hover-over showing the sequence-middle frame.
+    Clicks modify the object output to be used for
```
Comment (joshanderson-kw) [inline scripts/activity_detection_exploration/support_functions.py:98]: Missing end of sentence?

**Review by joshanderson-kw — APPROVED** [review]
LGTM, just one small comment.

**Comment by hdefazio** [comment]
Also general suggestion to add units to the Time label in the plots

File: scripts/activity_detection_exploration/support_functions.py (hunk 2 above)
Comment (Purg) [reply scripts/activity_detection_exploration/support_functions.py:98]: Good catch! Let me finish that.

File: scripts/activity_detection_exploration/support_functions.py (hunk 1 above)
Comment (Purg) [reply scripts/activity_detection_exploration/support_functions.py:132]: Valid. I suspect this was an artifact of my hacking things quickly before this was added as part of this function.

**Comment by Purg** [comment]
> Also general suggestion to add units to the Time label in the plots

Added.

**Pull request merged**